src/deps/c_ares.zig

```diff
@@ -200,39 +201,38 @@ pub const struct_hostent = extern struct {
     h_addr_list: [*c][*c]u8,
 
     pub fn toJSResponse(this: *struct_hostent, _: std.mem.Allocator, globalThis: *JSC.JSGlobalObject, comptime lookup_name: []const u8) JSC.JSValue {
-
-        // A cname lookup always returns a single record but we follow the common API here.
         if (comptime strings.eqlComptime(lookup_name, "cname")) {
-            if (this.h_name != null) {
-                const array = JSC.JSValue.createEmptyArray(globalThis, 1);
-                const h_name_len = bun.len(this.h_name);
-                const h_name_slice = this.h_name[0..h_name_len];
-                array.putIndex(globalThis, 0, JSC.ZigString.fromUTF8(h_name_slice).toJS(globalThis));
-                return array;
-            }
-            return JSC.JSValue.createEmptyArray(globalThis, 0);
-        } else {
-            if (this.h_aliases == null) {
+            // A cname lookup always returns a single record but we follow the common API here.
+            if (this.h_name == null) {
                 return JSC.JSValue.createEmptyArray(globalThis, 0);
             }
+            const array = JSC.JSValue.createEmptyArray(globalThis, 1);
```

Comment:
Use String.toJSArray, there are a surprising number of footguns when iteratively creating an array